content/_layouts/corebaseline.html.haml

```diff
@@ -13,6 +13,7 @@ layout: developersection
     lts = site._generated[:core_tiers].stableCores
     oldest_lts = lts[0]
 
+    # TOOD update
```

Comment:
```suggestion
    # TODO update
```

I can't believe it, the spellcheck found something useful 😆 

CC @halkeye

Reply:
:D it also found a few times SECURITY- was spelled wrong :)

Reply:
I have seen `-Dsome.option=flse` so this is nothing!

Reply:
probably not anymore, cause I added cargo-ci/typos github check.

Reply:
I'm sad the tooling didn't put an inline comment. So I might switch it to warnings-ng in the future.